internal/commands/relationship.go

```diff
@@ -94,11 +94,14 @@ var bulkDeleteCmd = &cobra.Command{
 	RunE:  bulkDeleteRelationships,
 }
 
-func writeRelationshipsFromArgsOrStdin(cmd *cobra.Command, args []string) error {
-	if ok := isArgsViaFile(os.Stdin) && len(args) == 0; ok {
-		return nil
+func StdinOrExactArgs(n int) cobra.PositionalArgs {
```

Comment:
nice

Reply:
yeah i should probably throw this into cobrautil, haha